staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -184,8 +185,20 @@ func AddInfof(ctx context.Context, format string, data ...interface{}) {
 func (rl *respLogger) Log() {
 	latency := time.Since(rl.startTime)
 	auditID := request.GetAuditIDTruncated(rl.req)
+
+	verb := rl.req.Method
+	if requestInfo, ok := request.RequestInfoFrom(rl.req.Context()); ok {
+		// If we can find a requestInfo, we can get a scope, and then
+		// we can convert GETs to LISTs when needed.
+		scope := metrics.CleanScope(requestInfo)
+		verb = metrics.CanonicalVerb(strings.ToUpper(verb), scope)
+		// FIX BEFORE MERGING: this if clause doesn't get hit. Fix or delete.
```

Comment:
Do you know why?

Reply:
Yeah, the filter chain ordering was wrong.